src/util.ts

```diff
@@ -37,7 +37,7 @@ export function nop() {
 export async function getTSConfig(
     rootDir: string, parmReadFilep?: ReadFileP): Promise<any> {
   const tsconfigPath = path.join(rootDir, 'tsconfig.json');
-  const json = await (parmReadFilep || readFilep)(tsconfigPath, 'utf8');
+  const json = await(parmReadFilep || readFilep)(tsconfigPath, 'utf8');
```

Comment:
wat? did the formatter produce this? Seems wrong. Nothing you can address though.

Reply:
Yep -- likely a bug in the formatter. I've modified the code a little to prevent this.